arwen/elrondapi/elrondei.go

```diff
@@ -586,34 +593,42 @@ func createAsyncCall(context unsafe.Pointer,
 	}
 }
 
+// TODO rename to setAsyncCallGroupCallback
 //export setAsyncContextCallback
 func setAsyncContextCallback(context unsafe.Pointer,
-	asyncContextIdentifier int32,
+	groupIDOffset int32,
 	identifierLength int32,
 	callback int32,
 	callbackLength int32,
 ) int32 {
-	host := arwen.GetVmContext(context)
-	runtime := host.Runtime()
+	// host := arwen.GetVmContext(context)
```

Comment:
Yes, this API function is completely disabled until the underlying logic is completed. It will be enabled before merging `feat/promises` into `master`.